style.css

```diff
@@ -587,18 +595,27 @@ div {
   background-size: 130px;
 }
 
-#mobile-menu {
-  height: 85vh;
-  border-bottom-right-radius: 2em;
-  border-bottom-left-radius: 2em;
-  margin-bottom: 2em;
-  mix-blend-mode: normal;
-  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+.header-nav-mobile {
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  color: #3a4a42;
+  border: 1px lightgrey solid;
+  border-radius: 5px;
```

Comment:
- [ ] It would be better for the background not to be scrollable when the mobile menu is open, this will create a better user experience  👍🏿